### PR TITLE
first iteration of trending page section

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Trending Page Section Remake

Before:
image

After :
image

- [x] code refactoring
- [x] responsive 

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/939

### CODE REVIEW

**Comment by Royal-lobster** [comment]
@OleanjiKingCode This is clean ! 

small thing, The last card is not correctly aligned with rest

CleanShot 2023-01-13 at 12 46 36@2x

File: src/components/Landing/TrendingWikis.tsx [hunk 1]
```diff
@@ -160,8 +164,453 @@ const TrendingWikis = ({ drops = [] }: { drops?: Wiki[] }) => {
         px={4}
         maxW="750"
       >{`${t('trendingWikisDescription')}`}</Text>
-      <Box maxW="1160px" mx="auto">
-        <Carousel
+
+      <Flex
```
Comment (Royal-lobster) [inline src/components/Landing/TrendingWikis.tsx:168]: We can divide this component to an other file since it exceeded 600 lines

**Review by Royal-lobster — APPROVED** [review]

File: src/components/Landing/TrendingWikis.tsx (hunk 1 above)
Comment (OleanjiKingCode) [reply src/components/Landing/TrendingWikis.tsx:168]: yes, this is just the first i did yeah istill havent finished thanks tho :)

Comment (Royal-lobster) [reply src/components/Landing/TrendingWikis.tsx:168]: Cool ! looks promising.

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
please re-inspect figma for the last card, doesn't look good yet, then the dark mode is yet to be done and other breakpoints are broken.

**Comment by Softdev1** [comment]
@OleanjiKingCode  can you work on the raised review from @Simplypheyie96  and @Royal-lobster  so that this issue can be closed down. well-done

**Comment by Royal-lobster** [comment]
@OleanjiKingCode looking good now ! one small thing, it misses out responsiveness for final 320 breakpoint. 

CleanShot 2023-01-18 at 12 43 03@2x

**Comment by Simplypheyie96** [comment]
The card is a little larger than the rest of the other cards.
Also the color used for the created/created by section of the card should be gray50. currently the color is not correct.
It doesn't look great on the md breakpoint yet and the last card should be aligned left.
for mobile, the card should also be equal in size with the rest. 
@OleanjiKingCode

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
this shouldn't be happening on the desktop view. the cards shouldn't break. the 3 should be on the same level.
image
the md breakpoint still have some issues with content spilling out of the cards, if there needs to be adjustment with design, do communicate the issue with me @OleanjiKingCode

**Comment by kesar** [comment]
I would highly recommend remove the date icon. AFAIK there is not other places across the site we do that and those cards have already a lot of information to add an extra icon. Whenever we add a new visual we need to think why that's not in created on wiki page, or in activity cards and wiki cards.

I would probably suggest to get rid of a 10 words description which highly doubt its gonna add any meaningful information to the user to click. Probably its better to reduce image and keep just title  and a line after that author. Worth to see how visually are the two options. If you check the original cmc widgets they are pretty clean and lean in information.

cc: @Simplypheyie96

**Comment by Simplypheyie96** [comment]
> I would highly recommend remove the date icon. AFAIK there is not other places across the site we do that and those cards have already a lot of information to add an extra icon. Whenever we add a new visual we need to think why that's not in created on wiki page, or in activity cards and wiki cards.
> 
> I would probably suggest to get rid of a 10 words description which highly doubt its gonna add any meaningful information to the user to click. Probably its better to reduce image and keep just title and a line after that author. Worth to see how visually are the two options. If you check the original cmc widgets they are pretty clean and lean in information.
> 
> cc: @Simplypheyie96

The issue is that the cmc page has a marketcap info in the cards alongside just the tokens, a little image and the token symbol, so to make ours different, since ours is a wiki, and we don't have symbols or marketcap info to add, the cards were looking really weird and empty, hence the decision to make it look similar to the wiki cards with the editor and date edited and a line of wiki details didn't really give any details which is why there is two. 

Also on cmc, the last card contains info for their articles and community post, but since ours is still a wiki, i don't think we should have the last card different from the others since it doesn't look cohesive and designing it with a carousel differently from the others doesn't look great either since it is the same layout.
i will show examples of what i mean by this in a few mins.

**Comment by kesar** [comment]
> to make it look similar to the wiki cards with the editor and date edited and a line of wiki details didn't really give any details which is why there is two

The problem is that there is not enough space for some of the data you want to hook, and we need to take decision on what to remove. We can't get same data in a wiki card w image, title, summary, editor & last edit than in a one row

For example, a row will look like 
Title: Otherside
Summary: "Otherside is a metaverse project launched by ..."

Do you think that summary will add any valuable information in order to decide to visit that wiki ? or we are adding that summary to look like a wiki card / to fill empty space?

We could try w perhaps removing editor & last time edit and try to see how it looks w a proper summary. But we need to reduce some of the content in that trending sections.


About the last carrousel, I agree it breaks the coherence a bit, but its a spot we might sell. Some sites reserve spaces like that one for google adwords or stuff like that, I guess its a tradeoff

**Comment by Simplypheyie96** [comment]
@kesar 
These are some of different iterations i came up with.
image

image

**Comment by Simplypheyie96** [comment]
@OleanjiKingCode we are going with this one 
Remember the last card will have a little animation where the cards keeps changing just like on cmc.
figma Link: https://www.figma.com/file/wosuJaHeV318k1ON82CZpo/%F0%9F%A7%A0-IQ.WIKI?node-id=14567%3A99660&t=RssuHq43rZBhziFa-11
image

**Comment by OleanjiKingCode** [comment]
Alright

**Comment by OleanjiKingCode** [comment]
A video of the design implementation @Simplypheyie96  @kesar 
https://user-images.githubusercontent.com/75235148/213553420-562c34b1-00a2-4cc9-8a35-9ad842739cbc.mp4

**Comment by kesar** [comment]
> A video of the design implementation @Simplypheyie96 @kesar https://user-images.githubusercontent.com/75235148/213553420-562c34b1-00a2-4cc9-8a35-9ad842739cbc.mp4

thx for the video, looks very promising 👍🏻

**Comment by Simplypheyie96** [comment]
> A video of the design implementation @Simplypheyie96 @kesar https://user-images.githubusercontent.com/75235148/213553420-562c34b1-00a2-4cc9-8a35-9ad842739cbc.mp4

Looks good!

**Comment by OleanjiKingCode** [comment]
Alright I'll get the correct data for the other cards 🚀

**Comment by Simplypheyie96** [comment]
> I dont know what data to put for featured wikis @Simplypheyie96
@kesar

**Comment by kesar** [comment]
the wikis that we show in homepage in hero + trending.. the ones that come from the api

**Comment by Softdev1** [comment]
for the featured wikis the data should be same with the wiki  on the hero session @kesar

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
Great work.
Please get rid of unnecessary space in card for other breakpoints. The desktop bp cards have some space so the cards can be aligned. 
image

**Comment by Simplypheyie96** [comment]
@OleanjiKingCode Here is a loading state for the section
figma: https://www.figma.com/file/wosuJaHeV318k1ON82CZpo/%F0%9F%A7%A0-IQ.WIKI?node-id=14838%3A94316&t=bAM4ICbmGM6B2Ruu-11

image

**Review by Simplypheyie96 — APPROVED** [review]
Great work!

**Review by Royal-lobster — APPROVED** [review]

**Review by Softdev1 — APPROVED** [review]

**Pull request merged**